pkg/build/make.go

```diff
@@ -38,6 +38,7 @@ func (m *MakeBuilder) Build() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("failed to get version: %v", err)
 	}
+	setReproducibilityEnv(m.RepoRoot)
 	cmd := exec.Command("make", target)
```

Comment:
+1 to Ben's suggestion
Let's set the env only for the specific cmd so we don't affect other things.